benchmarking/get_connected_devices.py

```diff
@@ -44,7 +44,6 @@
 
 class GetConnectedDevices(object):
     def __init__(self, **kwargs):
-        getLogger().setLevel(logging.FATAL)
```

Comment:
We need to set it to FATAL so that the value can be retrieved by the caller process. We don't want to output all the INFO fields if it is called in command line. So, need to set it in line 65. 

But with your change, this may be moot. I suggest to keep it however.

Reply:
I see. I think I will change it back once after this function is called.